plugins/modules/kibana_action.py

```diff
@@ -59,15 +57,21 @@ def format_config(self):
         'url': self.config['url'],
         'headers': self.config['headers']
       }
-
+    if self.action_type == 'Email':
+      return {
+        'from': self.config['sender'],
+        'hasAuth': self.config['auth'],
+        'host': self.config['host'],
+        'port': self.config['port']
+      }
+    
   def format_secrets(self):
     secrets = {}
     if self.action_type == 'webhook' and 'user' in self.secrets:
       secrets['user'] = self.secrets['user']
       secrets['password'] = self.secrets['password']
```

Comment:
Do we need to do something like this for the email connectors to support auth as well?